moped-database/migrations/1660258008607_insert_moped_milestones/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Delete added moped_milestone records
+DELETE FROM public.moped_milestones WHERE milestone_name = 'Draft Study Submitted';
+DELETE FROM public.moped_milestones WHERE milestone_name = 'Draft Preliminary Schematic Submitted';
+DELETE FROM public.moped_milestones WHERE milestone_name = 'Need to Assign Study';
```

Comment:
I tested the down migration, and if any of these milestones are in use on a project it will fail:

```sql
update or delete on table "moped_milestones" violates foreign key constraint "moped_project_milestone_milestone_id_fkey" on table "moped_proj_milestones". Key (milestone_id)=(54) is still referenced from table "moped_proj_milestones"
```

Question for the group—do we want to update the FK constraint on `moped_proj_milestones` to `ON UPDATE CASCADE` and `ON DELETE SET NULL` - ? I believe that's the best practice we've started to adopt elsewhere.


<img width="816" alt="Screen Shot 2022-08-12 at 10 42 20 AM" src="https://user-images.githubusercontent.com/14793120/184379825-ad06dd2d-a19d-4cb9-8a58-43209c1831eb.png">

Reply:
thanks @johnclary for pointing that out! i tested the down migration but i didn't think to do it with one of the milestones in use. i have added the migration files that update the fk as you recommended

Reply:
@johnclary i updated the fk relationship for moped_proj_milestones and restarted the cluster, added one of the new milestones to a project and ran the down migration and it still seems to be failing

do you have any recommendations on what to do?

<img width="1007" alt="Screen Shot 2022-08-16 at 3 13 47 PM" src="https://user-images.githubusercontent.com/35410637/184976765-147a4313-2560-4a39-a710-50f2016c3f1f.png">

Reply:
thanks for adding the constraints! i'll take a look 👀

Reply:
How did you test the down migration @tillyw? Did you run both of them in order?